src/test/java/de/zalando/aruha/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -140,6 +145,45 @@ public void whenPostWithNoSchemaSchemaThenReturn422() throws Exception {
     }
 
     @Test
+    public void whenPostWithUnknownPartitioningStrategyThenReturn422() throws Exception {
+        final EventType eventType = buildDefaultEventType();
+        final PartitionResolutionStrategy strategy = new PartitionResolutionStrategy("unknown_strategy", null);
+        eventType.setPartitionResolutionStrategy(strategy);
+
+        final Problem expectedProblem = Problem.valueOf(MoreStatus.UNPROCESSABLE_ENTITY,
+                "partitioning strategy does not exist: unknown_strategy");
+
+        postEventType(eventType)
+                .andExpect(status().isUnprocessableEntity())
+                .andExpect(content().contentType("application/problem+json"))
+                .andExpect(content().string(matchesProblem(expectedProblem)));
+    }
+
+    @Test
+    public void whenPostWithNullPartitioningStrategyNameThenReturn422() throws Exception {
+        final EventType eventType = buildDefaultEventType();
+        final PartitionResolutionStrategy strategy = new PartitionResolutionStrategy(null, null);
+        eventType.setPartitionResolutionStrategy(strategy);
+
+        final Problem expectedProblem = invalidProblem("partition_resolution_strategy.name", "may not be null");
+
+        postEventType(eventType)
+                .andExpect(status().isUnprocessableEntity())
+                .andExpect(content().string(matchesProblem(expectedProblem)));
+    }
+
+    @Test
+    public void whenPostWithKnownPartitioningStrategyThenReturn201() throws Exception {
```

Comment:
I would not like to do that, because we agreed that partitioning strategy should not be a required field. So I think default event-type should contain only the minimum amount of fields needed.